scripts/abins/instruments/directinstrument.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
I dont think we need this line anymore .

Reply:
Python is dead, long live Python